README.md

```diff
@@ -46,6 +46,17 @@ make install ## Required to have go installed in the machine
 stevedore --help ## Verify the installation
 ```
 
+### Use Docker image
+```
+alias stevedore="docker run -it -v ~/.config/stevedore:/root/.config/stevedore -v ~/.kube:/root/.kube -v $PWD:/workdir thecasualcoder/stevedore"
+stevedore --help
+```
+
+Note: In the above command, we are mounting
+1. `~/.config/stevedore` for stevedore to store the stevedore context
+2. `~/.kube` for stevedore to connect to kubernetes cluster
+3. `$PWD` for running plan/apply using yaml files in current working directory
+
```

Comment:
We could just give the docker command instead of an alias. Also the command is self-explanatory so the Note section looks redundant.

Reply:
yes make sense @arunvelsriram we can remove the alias and make the command self-explanatory

Reply:
We can remove the alias... I feel we can leave the `notes` as it might be helpful

Reply:
Okay. Lets have the notes